web/packages/core/.browserlistrc

```diff
@@ -0,0 +1,8 @@
+# Browsers that we support
+
+defaults
+chrome >= 87
+edge >= 87
+firefox >= 84
+iOS >= 14.1
+safari >= 14.1
```

Comment:
Not opposed to the addition of this file, but is this change related to this PR? I can't find any reference to it in the new code, though I may have missed it.

Reply:
autoprefixer uses it to decide which prefixes need to be added to support these browsers

Reply:
Decided to check the css to see if this is *actually* impactful, and it did catch at least one thing:

We used: `width: fit-content;`
It replaced it with:
```css
width: -moz-fit-content;
width: fit-content
```

Turns out `fit-content` isn't supported in firefox until 94

Reply:
On a related note, not sure why the generated CSS is adding `-moz-user-select` when full support for user-select was added in Firefox 69 and Firefox for Android 79, which is above our minimum supported version, but it's fine.

Reply:
-webkit-user-select is still useful though, as Safari uses that: https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#browser_compatibility